handlers/pingHandler.go

```diff
@@ -0,0 +1,11 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+)
+
+//PingHandler - to ping the server
+func PingHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Server is Running...")
```

Comment:
We're not using any formatting directives here, so `fmt.Fprint(w, "Server is running")` will suffice.

Reply:
Sure, I'll remove it...